proxies/live/apiproxy/proxies/default.xml

```diff
@@ -476,6 +476,17 @@
             <Step>
                 <Name>AssignMessage.ChangeGrantTypeToClientCrentials</Name>
             </Step>
+            <!-- An access token is generated to fetch all the scopes for filtering -->
+            <Step>
+              <Name>OAuthV2.TokenExchangeGenerateAccessToken</Name>
+          </Step>
+          <Step>
+              <Name>JavaScript.FilterUserRestrictedScopes</Name>
+          </Step>
+          <Step>
+              <Name>RaiseFault.VerifyAccessLevel</Name>
+              <Condition>(apigee.user_restricted_scopes = "")</Condition>
+          </Step>
             <Step>
                 <Name>OAuthV2.TokenExchangeGenerateAccessToken</Name>
```

Comment:
Are you passing the `<Scope>`apigee.user_restricted_scopes`<Scope>` to this policy?
This should be a different policy than the one you use to fetch all the scopes...